src/board/processor.ts

```diff
@@ -76,7 +76,8 @@ export class Processor extends EventEmitter<ProcessorEvents> {
    */
   public step(): boolean {
```

Comment:
The return value of `step()` is not used anymore.

Reply:
Good catch, changed the return type to `void`.